ISSUE_TEMPLATE.md

```diff
@@ -1,100 +1,123 @@
+### Is upstream commit [75b0cea7bf307f362057cc778efe89af4c615354 "ACPI: configfs: Disallow loading ACPI tables when locked down"](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=75b0cea7bf307f362057cc778efe89af4c615354) applied?
+-------------------------------------------------------------------------------
 `[your text here]`
 
-###### "Please specifically confirm that you add a vendor specific SBAT entry for SBAT header in each binary that supports SBAT metadata
-###### ( grub2, fwupd, fwupdate, shim + all child shim binaries )" to shim review doc ?
-###### Please provide exact SBAT entries for all SBAT binaries you are booting or planning to boot directly through shim
+-------------------------------------------------------------------------------
+### If shim is loading GRUB2 bootloader, what exact implementation of Secureboot in GRUB2 do you have? (Either Upstream GRUB2 shim_lock verifier or Downstream RHEL/Fedora/Debian/Canonical-like implementation)
+-------------------------------------------------------------------------------
 `[your text here]`
 
-##### Were your old SHIM hashes provided to Microsoft ?
-`[your text here]`
+-------------------------------------------------------------------------------
+### If shim is loading GRUB2 bootloader, and your previously released shim booted a version of grub affected by any of the following CVEs:
+* CVE-2020-14372
+* CVE-2020-25632
+* CVE-2020-25647
+* CVE-2020-27749
+* CVE-2020-27779
+* CVE-2021-20225
+* CVE-2021-20233
+* CVE-2020-10713
+* CVE-2020-14308
+* CVE-2020-14309
+* CVE-2020-14310
+* CVE-2020-14311
+* CVE-2020-15705
+* if you are shipping the shim_lock module: CVE-2021-3418
 
-##### Did you change your certificate strategy, so that affected by CVE-2020-14372, CVE-2020-25632, CVE-2020-25647, CVE-2020-27749,
-##### CVE-2020-27779, CVE-2021-20225, CVE-2021-20233, CVE-2020-10713,
-##### CVE-2020-14308, CVE-2020-14309, CVE-2020-14310, CVE-2020-14311, CVE-2020-15705 ( July 2020 grub2 CVE list + March 2021 grub2 CVE list )
-##### grub2 bootloaders can not be verified ?
+### Were old shims hashes provided to Microsoft for verification and to be added to future DBX updates?
+### Does your new chain of trust disallow booting old GRUB2 builds affected by the CVEs?
+-------------------------------------------------------------------------------
 `[your text here]`
 
-##### What exact implementation of Secureboot in grub2 ( if this is your bootloader ) you have ?
-##### * Upstream grub2 shim_lock verifier or * Downstream RHEL/Fedora/Debian/Canonical like implementation ?
+-------------------------------------------------------------------------------
+### Please specifically confirm that you add a vendor specific SBAT entry for SBAT header in each binary that supports SBAT metadata ( grub2, fwupd, fwupdate, shim + all child shim binaries ) to shim review doc.
```

Comment:
I'm not clear what's being referred to by "shim review doc" here. I think it's the readme, but I also think this is asking you to confirm that you've listed some SBAT stuff in your readme, while the readme says nothing about sbat.